components/styled/bookshelf.ts

```diff
@@ -0,0 +1,200 @@
+import tw from "tailwind-styled-components"
+
+const Container = tw.div`
+  w-full
+  grid
+  grid-cols-1
+  auto-rows-auto
+  mt-0
+  mx-auto
+  pt-1
+  pb-16
+  px-[84px]
+  gap-3
+`
+const LogoutContainer = tw.div`
+  flex
+  items-center
+  pt-2
+  px-4
+  pb-3
+  justify-end
+  gap-3
+`
+
+const NavBox = tw.nav`
+  px-6
+  pt-4
+  pb-7
+  border-solid
+  border-[1px]
+  border-[#f1f1f4]
+  rounded-[3px]
+  font-normal
+`
+
+const NavList = tw.ul`
+  list-none
+  p-0
+`
+
+const Link = tw.a`
+${(p) => (p.$active ? "bg-[#F1F1F4]" : "")}
```

Comment:
If you're seeing a pattern of these colors, it's time to put them into the `tailwind.config.js`